### PR TITLE
WCMS-472 Added groups to events

### PR DESCRIPTION
**Pull request by rift137**

### CODE REVIEW

File: lib/buweb/event.rb [hunk 1]
```diff
@@ -144,8 +146,10 @@ def as_indexed_json(options={})
       day: start_date.day,
       day_span: day_span,
       multiple_day_event: multiple_day_event?,
-      departments: departments.to_a.map(&:title),
+      departments_groups: departments.to_a.map(&:title),
```
Comment (jesilz) [inline lib/buweb/event.rb]: not sure we need to keep renaming this every time we add another relationship. what if we just index it as "organizations" ?

File: lib/buweb/event.rb [hunk 2]
```diff
@@ -144,8 +146,10 @@ def as_indexed_json(options={})
...
       department_slugs: departments.to_a.map(&:slug),
+      groups: groups.to_a.map(&:title),
```
Comment (jesilz) [inline lib/buweb/event.rb]: if we index departments and groups together do we need this?

File: lib/buweb/event.rb [hunk 3]
```diff
@@ -144,8 +146,10 @@ def as_indexed_json(options={})
...
+      group_slugs: groups.to_a.map(&:slug),
```
Comment (jesilz) [inline lib/buweb/event.rb]: and this?

File: lib/buweb/event.rb (hunk 1 above)
Comment (rift137) [inline lib/buweb/event.rb]: like so?
```
organizations: departments.to_a.map(&:title) + groups.to_a.map(&:title)
organizations_slugs: departments.to_a.map(&:slug) + groups.to_a.map(&:slug)
```

**Pull request merged**